### PR TITLE
feature: add pipeline build number into Pipeline list

### LINKED ISSUE #1299: Add build number into pipeline item component
### Clear and concise description of the problem

As a user of Woodpecker
I Want to see the build number into PipelineItem component
So that will show easily build numbers into pipeline list view.

### Suggested solution

I added hash with build number

image

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by segux**
Closes #1299

### CODE REVIEW

File: web/src/components/repo/pipeline/PipelineItem.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
     <div class="flex py-2 px-4 flex-grow min-w-0 <md:flex-wrap">
       <div class="<md:hidden flex items-center flex-shrink-0">
+        <span class="text-color px-2">#{{ pipeline.number }}</span>
```
[the remark this answers is not in the document]
Comment (segux) [reply web/src/components/repo/pipeline/PipelineItem.vue:22]: Not sure what are you talking about

[the remark this answers is not in the document]
Comment (segux) [reply web/src/components/repo/pipeline/PipelineItem.vue:22]: ah ok, yes not a problem can be  with -

[the remark this answers is not in the document]
Comment (segux) [reply web/src/components/repo/pipeline/PipelineItem.vue]: Done

File: web/src/components/repo/pipeline/PipelineItem.vue [hunk 2]
```diff
@@ -21,6 +21,7 @@
       <div class="<md:hidden flex items-center flex-shrink-0">
         <Icon v-if="pipeline.event === 'cron'" name="stopwatch" class="text-color" />
         <img v-else class="w-8" :src="pipeline.author_avatar" />
+        <span class="text-color px-2">#{{ pipeline.number }} - </span>
```
[the remark this answers is not in the document]
Comment (lafriks) [reply web/src/components/repo/pipeline/PipelineItem.vue:24]: And display as label/tag without dash ` - `

[the remark this answers is not in the document]
Comment (lafriks) [reply web/src/components/repo/pipeline/PipelineItem.vue:24]: Same style like we have in secret list for event types

**Comment by 6543** [comment]
On UI stuff a screenshot is nice ;)

**Review by 6543 — COMMENTED** [review]
Lgtm

**Pull request merged**